index.css

```diff
@@ -895,35 +912,131 @@ ul a {
     font-size: 32px;
     font-weight: 400;
     color: #091e42;
+    margin-bottom: 0;
   }
 
   .section2-paragraph {
     font-family: "Inter", sans-serif;
     line-height: 24px;
+    margin: 0;
     color: #091e42;
+    width: 82%;
+  }
+
+  .see__project {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    font-weight: 700;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    padding: 10px 0;
+    background: #ff6b00;
+    border: none;
+    width: 30%;
+    color: #fff;
   }
 
   .section-cards {
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
+    margin: 4%;
+    width: 100%;
```

Comment:
- [ ] Kindly delete this `margin: 4%;` because it is adding extra space on the document,kind of overflow, I recorded a video to show it: 

https://user-images.githubusercontent.com/77038610/149617118-e585071b-7b0a-45e3-852d-c52991e03510.mp4

> you can change `margin: 4%` to `margin-left: 4%`; 😉